reverse_proxy.go

```diff
@@ -518,12 +518,18 @@ func (p *ReverseProxy) WrappedServeHTTP(rw http.ResponseWriter, req *http.Reques
 	// Circuit breaker
 	breakerEnforced, breakerConf := p.CheckCircuitBreakerEnforced(p.TykAPISpec, req)
 
+	// set up TLS certificates for upstream if needed
+	var tlsCertificates []tls.Certificate = nil
```

Comment:
Do you added `= nil` for expressivity or it is needed somehow?

Reply:
it is not needed as zero value is already nil, I think I added it just looking at " = nil" assignment in  https://github.com/TykTechnologies/tyk/pull/1342/files#diff-da06648202ab00313675c5150a92bffeL524